euler-maven-1/src/main/java/com/ceritoxi/Math.java

```diff
@@ -0,0 +1,23 @@
+package com.ceritoxi;
+
+public class Math {
+	
+	public static boolean isDividableByXorY(int dividend, int numberX, int numberY) {
+		if (numberX == 0 && numberY == 0) return false ;
+		if (numberX != 0 && dividend % numberX == 0) return true;
```

Comment:
the  zero check already happened in the previous line. Add further logic in an "else if" clause, also its redundant to use two if, you can do it in one go. such as:
```
} else if ( (dividend % numberX == 0) || (dividend % numberY == 0)) {
 return true;
}

Reply:
Well the code that you wrote is flawed, but indeed the first line of my code if completely useless now that I think about it. So I will simply remove it. With that change the else if is no longer needed.